kategory/src/test/kotlin/kategory/data/EvalTest.kt

```diff
@@ -8,7 +8,10 @@ import org.junit.runner.RunWith
 class EvalTest : UnitSpec() {
     init {
 
-        testLaws(MonadLaws.laws(Eval))
+        testLaws(MonadLaws.laws(Eval, object : Eq<HK<Eval.F, Int>> {
```

Comment:
We should have this instance in the core not just in tests and there should be an instance for each one of the datatypes we implement.